backend/infrahub/computed_attribute/tasks.py

```diff
@@ -158,6 +160,57 @@ async def trigger_update_python_computed_attributes(
         )
 
 
+@flow(
+    name="process_computed_attribute_value_jinja2",
+    flow_run_name="Update value for computed attribute {attribute_name}",
+)
+async def update_computed_attribute_value_jinja2(
+    branch_name: str, obj: CoreNode, attribute_name: str, template_value: str
```

Comment:
I think we could use something more descriptive than "obj" here.

Reply:
Initially I used `node` but it's currently a reserved keyword for the batch so it wouldn't work